### PR TITLE
Timespinner: rename some locations for clarity.

### PR DESCRIPTION
**Pull request by alwaysintreble**
Adjusted a lot of locations to be more clear on where they're located for when hints are received. Put bat jump chest and royal guard tiny room in logic when the player has talaria + wheel.

### CODE REVIEW

**Comment by Berserker66** [comment]
This'll be waiting for @JarnoWesthof approval.

**Comment by JarnoWesthof** [comment]
Id like some locations changed, and also need to fix some things as atm it will break client compatibility, i will go into bigger details later, but just putting this comment here so it hopefully wont be merged like this

File: worlds/timespinner/PyramidKeys.py
Comment (JarnoWesthof) [inline worlds/timespinner/PyramidKeys.py:9]: We got to rename this back to GateMilitairyGate, as its part of contract between client and server, to fix this type we would first need to change the client to support both versions and then we can change the server side

File: worlds/timespinner/LogicMixin.py
Comment (JarnoWesthof) [inline worlds/timespinner/LogicMixin.py:19]: this should use self.has_all when checking two specific items

File: worlds/timespinner/Locations.py
Comment (JarnoWesthof) [inline worlds/timespinner/Locations.py:27]: The names are still a bit under discussion ill have a look at them later

**Review by JarnoWesthof — CHANGES_REQUESTED** [review]

File: worlds/timespinner/PyramidKeys.py
Comment (alwaysintreble) [reply worlds/timespinner/PyramidKeys.py]: reverted

**Comment by JarnoWesthof** [comment]
There are some conflicts as i already let the logic changes be merged in 2.0 so they are already live

**Pull request closed without merging**
